tests/InvokableAnyObject.php

```diff
@@ -0,0 +1,12 @@
+<?php
+declare(strict_types=1);
+
+namespace Fig\EventDispatcher;
+
+class InvokableAnyObject
+{
+    public function __invoke(object $event)
+    {
+        // do nothing
+    }
+}
```

Comment:
End of file is missing.
```suggestion
}

```

Reply:
Fixed via commit - something blocks me from "accepting" the suggestion.